packages/flutter/lib/src/widgets/drag_target.dart

```diff
@@ -358,6 +358,11 @@ class Draggable<T extends Object> extends StatefulWidget {
 }
 
 /// Makes its child draggable starting from long press.
+///
+/// See also:
+///
+///  * [Draggable]
```

Comment:
there should be an explanation here telling me why I should also look at "draggable".

Reply:
Good catch, I will update both

Reply:
Done.